ui/src/components/job/JobView.js

```diff
@@ -4,14 +4,52 @@ import { useDispatch, useSelector } from 'react-redux';
 import { jobRequested } from './jobReducer';
 import { useParams } from 'react-router';
 import AccountBalanceOutlinedIcon from '@material-ui/icons/AccountBalanceOutlined';
+import Timeline from '@material-ui/lab/Timeline';
+import TimelineItem from '@material-ui/lab/TimelineItem';
+import TimelineSeparator from '@material-ui/lab/TimelineSeparator';
+import TimelineConnector from '@material-ui/lab/TimelineConnector';
+import TimelineContent from '@material-ui/lab/TimelineContent';
+import TimelineOppositeContent from '@material-ui/lab/TimelineOppositeContent';
+import TimelineDot from '@material-ui/lab/TimelineDot';
+import { makeStyles } from '@material-ui/core/styles';
+import PublishIcon from '@material-ui/icons/Publish';
+import SaveIcon from '@material-ui/icons/Save';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import EditIcon from '@material-ui/icons/Edit';
+import BackupIcon from '@material-ui/icons/Backup';
+import HighlightOffIcon from '@material-ui/icons/HighlightOff';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+const useStyles = makeStyles((theme) => ({
+    paper: {
+        padding: '6px 16px',
+    },
+    secondaryTail: {
+        backgroundColor: theme.palette.secondary.main,
+    },
+}));
+
+const event2icon = {
+    'UPLOADED': function display() { return (<PublishIcon />);},
+    'VALIDATING': function display() { return (<CheckCircleOutlineIcon />);},
+    'STAGING': function display() { return (<CheckCircleOutlineIcon />);},
+    'STAGED': function display() { return (<SaveIcon />);},
+    'EDITING':function display() { return(<EditIcon />);},
+    'INGESTING':function display() { return (<BackupIcon />);},
+    'CORRECTING':function display() { return (<CheckCircleOutlineIcon />);},
+    'INGESTED': function display() { return (<CheckCircleOutlineIcon />);},
+    'CORRECTED': function display() { return(<CheckCircleOutlineIcon />);},
+    'DELETED': function display() { return (<HighlightOffIcon />);},
+    'ABANDONED': function display() { return (<CheckCircleOutlineIcon />);},
+    'ERROR': function display() { return (<ErrorOutlineIcon />);}
+};
 
 const JobView = () => {
     const dispatch = useDispatch();
     const { id } = useParams();
     const job = useSelector(state => state.job.currentJob);
-    //const errors = useSelector(state => state.job.errors);
+    console.log(job);
```

Comment:
Remove console.log

Reply:
console.log are breaking hte build it should be gone now